File: core/src/com/bosowski/oslark/Oslark.java
package com.bosowski.oslark;

import com.badlogic.gdx.Application;
import com.badlogic.gdx.ApplicationAdapter;
import com.badlogic.gdx.Game;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.Screen;
import com.badlogic.gdx.assets.AssetManager;
import com.badlogic.gdx.maps.tiled.TiledMap;
import com.badlogic.gdx.maps.tiled.TiledMapTileLayer;
import com.badlogic.gdx.maps.tiled.TiledMapTileSet;
import com.badlogic.gdx.maps.tiled.TmxMapLoader;
import com.badlogic.gdx.maps.tiled.tiles.StaticTiledMapTile;
import com.bosowski.oslark.main.Assets;
import com.bosowski.oslark.screens.LoginScreen;

import java.io.File;

public class Oslark extends Game {
	
	@Override
	public void create () {
		// Set Libgdx log level
		Gdx.app.setLogLevel(Application.LOG_DEBUG);

		// Load assets
		Assets.instance.init(new AssetManager());

		setScreen(new LoginScreen(this));
	}

}

File: OslarkGame/core/src/com/bosowski/oslark/Oslark.kt
package com.bosowski.oslark

import com.badlogic.gdx.Application
import com.badlogic.gdx.Game
import com.badlogic.gdx.Gdx
import com.badlogic.gdx.assets.AssetManager
import com.badlogic.gdx.math.Vector2
import com.bosowski.oslark.gameObjects.prefabs.monsters.Skeleton
import com.bosowski.oslark.gameObjects.prefabs.playerClasses.Knight
import com.bosowski.oslark.screens.GameScreen
import com.bosowski.oslark.screens.LoginScreen


class Oslark : Game() {

  override fun create() {
    // Set Libgdx log level
    Gdx.app.logLevel = Application.LOG_DEBUG

    // Load assets
    Assets.init(AssetManager())

    setScreen(LoginScreen(this))



//    val demon = Demon(position = Vector2(1f,1f))
//    demon.instantiate()
  }
}
